### PR TITLE
[zh]translate docs: karmada-agent

### PR DESCRIPTION
**Pull request by justedennnnn**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/components/karmada-agent.md [hunk 1]
```diff
@@ -2,19 +2,16 @@
 title: karmada-agent
 ---
 
+### 简介
```
Comment (SAMZONG) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/components/karmada-agent.md:5]: pls update the first headtag `###` to `##`, also english version.

Comment (justedennnnn) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/components/karmada-agent.md]: Done， and thanks for review

**Comment by Poor12** [comment]
This file is auto generated by scripts. It is best not to modify this.

**Comment by justedennnnn** [comment]
> This file is auto generated by scripts. It is best not to modify this.
okay,  i will close this PR.

**Pull request closed without merging**